recipes/libsafec/all/conanfile.py

```diff
@@ -99,7 +105,17 @@ def generate(self):
             f"--enable-strmax={self.options.strmax}",
             f"--enable-memmax={self.options.memmax}",
         ]
-        tc.generate()
+
+        if is_apple_silicon:
+            tc.configure_args += [
+                "--disable-hardening",
+            ]
+            # create a universal binary
+            tc_env.define("CC", "clang -arch arm64 -arch x86_64")
+            tc_env.define("CXX", "clang -arch arm64 -arch x86_64")
+            tc_env.define("CPP", "clang -E")
+            tc_env.define("CXXCPP", "clang -E")
+        tc.generate(tc_env)
```

Comment:
At first, I'd like to see the logs resulting from running it with/without that custom env:

```suggestion
        if is_apple_silicon:
            tc.configure_args += [
                "--disable-hardening",
            ]
        tc.generate()
```
After seeing those logs, we can discuss how to make this part avoid using this custom env and try to use only extra C/C++ flags (if possible, for sure). In these cases, it's better to try to use the `tc.extra_cflags`/`tc.extra_cxxflags` attribute to simplify the logic.

Reply:
Hey @franramirez688,

Thanks for the quick review!  I have removed the custom environment used `tc.extra_cflags` and `tc.extra_cxxflags`.  These extra flags are what I was originally looking for.

Let me know if you still want the logs with the custom environment.

Thanks,
Leo